packages/react-art/Rectangle.js

```diff
@@ -7,8 +7,4 @@
  * @flow
  */
 
-'use strict';
-
-const Rectangle = require('./npm/Rectangle');
-
-module.exports = Rectangle;
+export {default} from './npm/Rectangle';
```

Comment:
How does this work? Aren’t they still CJS?

Reply:
Yea, for now. But once ART switches to ES modules we can switch the inner implementation too.